notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p>Out of curiosity - can we use smallest variant and list in comments other options?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The thext encoder is the bottleneck anyway, but it is reasonable to default to the smallest models anyway</p>